docs/sources/guides/whats-new-in-v7-2.md

```diff
@@ -79,4 +81,11 @@ See [upgrade notes]({{< relref "../installation/upgrading.md" >}}).
 
 Check out [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md) for a complete list of new features, changes, and bug fixes.
 
+# News from other parts of the Grafana ecosystem
+
+## ADX (Azure Data Explorer) plugin
+In collaboration with Microsoft we have started to improve the usability of the ADX datasource plugin. The goal is to make it easier for users, regardless on their previous knowledge of writning KQL (Kusto Query Language) queries, to work with and visualize their data.
```

Comment:
@daniellee is it okay to mention MS here?

Reply:
@mckn absolutely fine to mention that we are collaborating with MS. I think I would phrase this paragraph differently. The ADX plugin isn't new - it's the visual query builder that is new. When using the raw mode then queries have been performant if you wrote a good query so not sure that part is totally correct either.

(Also, spelling `regardless on their previous knowledge of writning KQL` -> `regardless of their previous knowledge of writing KQL`)

Reply:
Awesome feedback! Not sure I am following on the thinking of the datasource plugin being new. 

"In collaboration with Microsoft we have started to improve the usability of the ADX datasource plugin. The goal is to make it easier for users, regardless of their previous knowledge of writing KQL (Kusto Query Language) queries, to work with and visualize their data.

This is the first step towards a more user friendly and flexible datasource plugin. More updates to come. Stay tuned for more updates at [ADX datasource plugin](https://github.com/grafana/azure-data-explorer-datasource)"

Would this work?

Reply:
@mckn if I'm reading this, it sounds like we have built a new plugin for ADX rather than telling that we built a new feature - the visual query builder.

Reply:
Anyway, good for enough now.

Reply:
@daniellee aha, that wasn't my intention. I will try to change it a bit then. Give me a couple of minutes :)